sweep/walletsweep.go

```diff
@@ -47,117 +51,78 @@ func (p FeePreference) String() string {
 }
 
 // Estimate returns a fee rate for the given fee preference. It ensures that
-// the fee rate respects the bounds of the relay fee and the specified max fee
-// rates.
-//
-// TODO(yy): add tests.
+// the fee rate respects the bounds of the relay fee and the max fee rates, if
+// specified.
 func (f FeePreference) Estimate(estimator chainfee.Estimator,
 	maxFeeRate chainfee.SatPerKWeight) (chainfee.SatPerKWeight, error) {
 
-	// Get the relay fee as the min fee rate.
-	minFeeRate := estimator.RelayFeePerKW()
+	var (
+		feeRate chainfee.SatPerKWeight
+		err     error
+	)
 
+	switch {
 	// Ensure a type of fee preference is specified to prevent using a
 	// default below.
-	if f.FeeRate == 0 && f.ConfTarget == 0 {
```

Comment:
There is really no default below

Reply:
what do you mean?

Reply:
I was pointing out that there was no default case in the switch statement but rereading it now, I think that was the intention.